sw/device/silicon_creator/mask_rom/data/mask_rom_testplan.hjson

```diff
@@ -13,6 +13,54 @@
   name: "mask_rom"
 
   testpoints: [
+    // Shutdown: Error reporting
+    {
+      name: mask_rom_e2e_shutdown_output
+      desc: '''Verify that mask ROM can properly report errors over UART.
+
+            - Attempt to boot without a valid ROM_EXT.
+            - Verify that we can receive 28 characters (LCV:xxxxxxxx\r\nBFV:xxxxxxxx\r\n) of error
+              output in all life cycle states where Ibex is enabled, i.e. TEST_UNLOCKED*, PROD,
+              PROD_END, DEV, and RMA.
+                - BFV should be 0x142500d for all life cycle states.
+                - See the table below for the expected LCV value.
+
+            | LC State |     LCV    | OTP LIFE_CYCLE state | OTP LIFE_CYCLE count |
+            |:--------:|:----------:|:--------------------:|:--------------------:|
+            |   TEST   | 0x02108421 |   "TEST_UNLOCKED0"   |           5          |
+            |   DEV    | 0x21084210 |        "DEV"         |           5          |
+            |   PROD   | 0x2318c631 |        "PROD"        |           5          |
+            | PROD_END | 0x25294a52 |      "PROD_END"      |           5          |
+            |   RMA    | 0x2739ce73 |        "RMA"         |           5          |
+            '''
+      tags: ["verilator", "dv", "fpga", "silicon"]
+      milestone: V2
+      tests: ["mask_rom_e2e_shutdown_output"]
```

Comment:
This list should be empty right now since the test doesn't exist yet right?

Reply:
I was hoping to add it as part of this PR but we'll see :)

Reply:
Removed.